include/NovelRT.Interop/Graphics/NrtGraphicsTypedefs.h

```diff
@@ -0,0 +1,7 @@
+typedef struct
+{
+    int32_t r;
+    int32_t g;
+    int32_t b;
+    int32_t a;
+} NrtRGBAColour;
```

Comment:
Why are we using a `128 bits per pixel/32 bits per component` format while the functions are normalizing assuming a `32 bits per pixel/8 bits per component` format?

I'd expect this is either using `uint8_t x;` or `uint32 x : 8;` for each component (I believe the latter is more efficient from an ABI perspective).

Reply:
If this is intentionally 32-bits per component, then I'd largely expect the full range gets utilized as this would correspond to `VK_FORMAT_R32G32B32A32_SINT` (but even then, I'd probably expect `VK_FORMAT_R32G32B32A32_UINT` is better and these should be unsigned).

But, I'd also expect wanting 32-bits per component for integer formats is generally undesirable. For such a format, `float` is more typical.

Reply:
We only currently use `RGBAColour` for the background clear, which uses the scalar methods which are 32 bit floats for each value in the `VkClearValue` object.

The size change should be fine. :)